src/styles.scss

```diff
@@ -329,6 +329,16 @@ body {
   .view-container {
     background-color: $white;
     padding: 1rem;
+    mat-radio-group {

```

Comment:
This may be a useful app wide style change, but I think for now let's stick to changing meetups and we can reevaluate and make a new issue if needed.